test/crypto/olm_device_test.py

```diff
@@ -0,0 +1,69 @@
+from copy import deepcopy
+
+from matrix_client.client import MatrixClient
+from matrix_client.crypto.olm_device import OlmDevice
+
+HOSTNAME = 'http://example.com'
+
+
+class TestOlmDevice:
+    cli = MatrixClient(HOSTNAME)
+    user_id = '@user:matrix.org'
+    device_id = 'QBUAZIFURK'
+    device = OlmDevice(cli.api, user_id, device_id)
+    signing_key = device.olm_account.identity_keys()['ed25519']
+
+    def test_sign_json(self):
+        example_payload = {
+            "name": "example.org",
+            "unsigned": {
+                "age_ts": 922834800000
+            }
+        }
+        saved_payload = deepcopy(example_payload)
+
+        signed_payload = self.device.sign_json(example_payload)
+        signature = signed_payload.pop('signatures')
+        # We should not have modified the payload besides the signatures key
+        assert example_payload == deepcopy(saved_payload)
+        key_id = 'ed25519:' + self.device_id
+        assert signature[self.user_id][key_id]
+
+    def test_verify_json(self):
+        example_payload = {
+            "test": "test",
+            "unsigned": {
+                "age_ts": 922834800000
+            },
+            "signatures": {
+                "@user:matrix.org": {
+                    "ed25519:QBUAZIFURK": ("WI7TgwqTp4YVn1dFWmDu7xrJvEikEzAbmoqyM5JY5t0P"
```

Comment:
For my sanity, could you add a `+` at EOL here? Implicitly concatenated strings weird me out.

Reply:
Alright. Fair enough.